pages/inspector/connect-inspector-to-gtm.mdx

```diff
@@ -1,14 +1,74 @@
+
+**_Reminders:_** 
+- The Inspector SDKs are designed with flexibility in mind, so you should always be able to add observability to the source of your pipelines.
+- Avo Inspector operates on the environment level, please keep it in mind and create separate tags for development, staging and production data.
+
+## Getting Started
+Avo Inspector requires two core steps to add observability:
+1. Initializing the Inspector
+2. Sending events to Inspector
+
+Both steps have installation options available to you to support different configurations. Which ever path you take, your data observability journey starts with creating an Avo source because this is where you create source specific API keys and can get the initialization code. If you haven't created a source yet, follow the docs to [create and set up your GTM source](/inspector/configuring-inspector-sources).
+
+## Inspector Client side GTM integration (beta)
+
+There are two options to install Inspector via Google Tag Manager. You can either import our GTM recipe to add observability to all data layer events, or use our Inspector function to add observability to individual events.
+
+<Callout type="info" emoji="💡">
+We recommend starting by installing the Inspector on your development or testing GTM container and connect it to the Inspector dev environment. That is both so your Inspector production environment is not contaminated by testing data and because the Inspector development environment is designed to handle low data volumes real time while the production environment is optimized for large amount of production data and has 1 - 2 hour delay.
+</Callout>
+
+### Importable GTM recipe
+
+To add observability to all data layer events with our importable GTM recipe [in under 90 seconds](https://www.loom.com/share/c659b857d4d947729683bf11691f441e), follow the steps below:
+1. [Download our GTM recipe here]()
+2. Import the recipe into your GTM container. 
+3. Add the source specific Inspector API key. 
+4. When testing, adjust the the Avo Inspector Init tag by changing the 'prod' value to 'dev' on the inspector.__ENV__ key.
+5. Publish the container.
+6. Visit the Avo Inspector dashboard in one hour to see
+7. Consider creating a filtered issues view for the data bugs you care about. [learn more here](/inspector/inspector-issues-view).
+
+<Callout type="info" emoji="💡">
```

Comment:
@thoragudf think we should move this to the top of this section. inItialisation is like a step one. Ideally we have a whole page section on Initialisation, but easy to get into a rabbit whole there, so kept this GTM focused. But bumping it to the top helps skimmers know that this is baked into the recipe.

Reply:
I mean sat under this subheader https://docs-git-inspector-web-gtm-beta.avo-dev.app/docs/inspector/connect-inspector-to-gtm#importable-gtm-recipe